Godnoken/Sprint-Challenge--JavaScript
# JavaScript Sprint Challenge

## What does this PR do?
JavaScript Sprint Challenge I Challenge solutions & Stretch goals
​
## Description of tasks completed
- Changed the README.md so it now contains answers to the questions included
- Implemented the requested code to each .js file to match the description of the assignments
- Implemented a subclass to Cuboid with two new methods (Stretch)
​
## How can this be tested?
1. Clone the project.
2. Open up the .js files in your favourite code editor.
3. Check for changes in the console after you've ran the code.
4. Alternatively, open up index.html and look inside your browsers console for changes.

## Relevant screenshot
![image](https://user-images.githubusercontent.com/17499594/67081358-44e93280-f197-11e9-9e88-48f5232013ed.png)

![image](https://user-images.githubusercontent.com/17499594/67081405-5d594d00-f197-11e9-9b5e-3b06f06fc5e6.png)
